app/vulncode_app/models.py

```diff
@@ -26,38 +26,35 @@
 
 
 class Vulnerability(models.Model):
-	vulnerability_id = models.AutoField(primary_key=True)
-	summary = models.TextField(max_length=50)
-	cvss = models.FloatField(max_length=50)
+    summary = models.TextField(max_length=50, help_text="Summary of the vulnerability")
+    cvss = models.FloatField(max_length=50, help_text="CVSS Score")
 
 
 class VulnerabilityReference(models.Model):
-	vulnerability_id = models.ForeignKey('Vulnerability')
-	source = models.CharField(max_length=50)
-	reference_id = models.CharField(max_length=50)
-	url = models.URLField(max_length=50)
+    vulnerability = models.ForeignKey('Vulnerability')
+    source = models.CharField(max_length=50, help_text="Source's name eg:NVD")
+    reference_id = models.CharField(max_length=50, help_text="Reference ID, eg:CVE-ID")
+    url = models.URLField(max_length=1024, help_text="URL of Vulnerability data")
 
 
 class ImpactedPackage(models.Model):
-	vulnerability_id = models.ForeignKey('Vulnerability')
-	package_id = models.ForeignKey('Package')
+    vulnerability_id = models.ForeignKey('Vulnerability')
```

Comment:
As you did for `VulnerabilityReference.vulnerability`, you should remove the `_id` suffix. For both fields `vulnerability_id` and `package_id`